portfolio/src/main/webapp/style.css

```diff
@@ -52,7 +84,7 @@
 
 /* Style the header links */
 .header a {
-  float: left;
+  /* float: left; */
```

Comment:
?

Reply:
OH OOPS

Reply:
I think I was just playing around to see if the float made any impact and forgot to delete it -- fixed!